### ISSUE TITLE
CVar hideHelptips

### ISSUE THREAD

**Issue by voidzone** [Stanzilla/AdvancedInterfaceOptions#63]
https://github.com/Gethe/wow-ui-source/blob/f0084386950fe3dc31a1d61de33b364e268cf66b/Interface/SharedXML/HelpTip.lua#L145

**Comment by semlar** [comment]
If this is intended to be a bug report, please describe the issue.

**Comment by voidzone** [comment]
Just noticed it wasn't in the cvar browser.

**Comment by Ketho** [comment]
It seems to be either Blizzard forgot to register it as an actual cvar or they intended it to be used via `C_CVar.RegisterCVar("hideHelptips", 1)`. Anyway thanks for the tip, it will be a good addition to HideTutorial 👍

**Comment by voidzone** [comment]
Yeah, also looks like it doesn't get saved if you exit the game, but will persist though logouts.
